.github/workflows/generic-deb-workflow.yml

```
@@ -46,8 +54,19 @@ jobs:
           GITHUB_TOKEN: '${{ secrets.auto_commit_pwd }}'
           MERGE_MSG: "Branch was auto-updated."
 
+  Pre-commit:
+    if: ${{ inputs.deploy == 'false' }}
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.10'
+    - uses: pre-commit/action@v2.0.3
+
   Pack:
     runs-on: ubuntu-20.04
```

why Pack does not depend on Pre-commit ?
```suggestion
    runs-on: ubuntu-20.04
    needs: [Pre-commit]
```